packages/deployment/files.js

```diff
@@ -1,70 +1,76 @@
-import util from 'util';
-import { resolve, basename, dirname } from 'path';
-import {
-  chmod as rawChmod,
-  exists as rawExists,
-  mkdir as rawMkdir,
-  readFile as rawReadFile,
-  stat as rawStat,
-  writeFile as rawWriteFile,
-  readdir as rawReaddir,
-  write as rawWrite,
-  close as rawClose,
-  unlink as rawUnlink,
-  rename as rawRename,
-} from 'fs';
+import { promisify } from 'util';
 import { Readable } from 'stream';
-import { open as tempOpen } from 'temp';
 import chalk from 'chalk';
```

Comment:
Not sure if you care, but `'chalk'` is also ambient.  It accesses `process.env`.  Perhaps you'd like to pretend you never saw this?

Reply:
These are not the modules you're looking for.  We'll get to it later.